src/assets/translations/en/main.json

```diff
@@ -1811,6 +1811,12 @@
     "ledgerOpenApp": {
       "title": "Open the %{appName} App",
       "description": "To connect this chain, you will need to open the Ledger app on your device."
+    },
+    "errors": {
+      "accountMetadataFailedToFetch": {
+        "title": "Account Metadata Fetch Failed",
```

Comment:
This seems like quite a low-level messaging, but given the root cause is eventually going to get fixed, fine with that!